src/registry/metadataRegistry.json

```diff
@@ -4878,6 +4879,14 @@
       "directoryName": "apiNamedQueries",
       "inFolder": false,
       "strictDirectoryName": false
+    },
+    "externalStoragePrvdConfig": {
+      "id": "externalStoragePrvdConfig",
+      "name": "ExternalStoragePrvdConfig",
+      "suffix": "ExternalStoragePrvdConfigSet",
```

Comment:
I believe this needs to be lower case. I get a test failure when running `yarn test` on your fork 
```bash
  1) ConnectionResolver
       resolve
         should resolve parent and child components:
     RegistryError: Missing metadata type definition in registry for id 'ExternalStoragePrvdConfig'.
```